webextensions/api/declarativeNetRequest.json

```
@@ -1107,6 +1107,82 @@
               }
             }
           },
+          "excludedResponseHeaders": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "128"
```

To header, values and excludedValues, emphasize that previous versions recognized them but were not effective. This is important because extensions relying on this could match more than intended on older versions.

```suggestion
                  "version_added": "128",
                  "notes": "From Chrome 121 until Chrome 127, this condition was recognized but not applied."
```

Version 121 is based on the version that introduced this:

- https://source.chromium.org/chromium/chromium/src/+/012df41d1605ba32865b102b17f247ecaf56de2e
- https://source.chromium.org/chromium/chromium/src/+/012df41d1605ba32865b102b17f247ecaf56de2e:chrome/VERSION